src/views/CTLViewsHoC.jsx

```diff
@@ -82,6 +80,9 @@ const CTLViewsHoC = (WrappedView) => {
 					navigate(
 						`${CodeOrPurlPath({ codeOrPurl: prettyUrlName })}${queryString}`
```

Comment:
Looking at this now, I think you might need the second parameter for `navigate` `{ replace: true }` should the user hit the back button after redirection. I missed this on the initial version.

Reply:
Ok will do